From 951230ccd2ec050ec59325bcf61562828fff9ee3 Mon Sep 17 00:00:00 2001
From: Guanhua Gao <guanhua.gao@nxp.com>
Date: Fri, 1 Jun 2018 15:35:22 +0800
Subject: [PATCH 543/767] Revert "drivers: flag buses which demand DMA
 configuration"

Revert "drivers: flag buses which demand DMA configuration" to not
change the framework of kernel.

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/of/device.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/of/device.c b/drivers/of/device.c
index ee1f900ba6ce..e1b49b51ebf6 100644
--- a/drivers/of/device.c
+++ b/drivers/of/device.c
@@ -10,7 +10,9 @@
 #include <linux/module.h>
 #include <linux/mod_devicetable.h>
 #include <linux/slab.h>
+#include <linux/pci.h>
 #include <linux/platform_device.h>
+#include <linux/amba/bus.h>
 
 #include <asm/errno.h>
 #include "of_private.h"
-- 
2.17.0

